.gitignore

```diff
@@ -1,3 +1,130 @@
-node_modules
-/tsc.help.txt
-/rollup.help.txt
+# Logs
```

Comment:
The gitignore for nodejs projects as found here: https://github.com/github/gitignore/blob/main/Node.gitignore